Cargo.toml

```diff
@@ -348,6 +348,9 @@ ios_simulator = ["bevy_internal/ios_simulator"]
 # Enable built in global state machines
 bevy_state = ["bevy_internal/bevy_state"]
 
+# Enable function reflection
+reflect_functions = ["bevy_internal/reflect_functions"]
```

Comment:
Thoughts on naming? I originally had `function_reflection`, but I switched it to `reflect_functions` since it sounds a bit more like the `bevy_reflect/functions`.

I'm also trying to think about potential future features. For example, we have a `bevy_reflect/documentation` feature. Using this naming strategy, it could be `reflect_documentation`. And so on.

Reply:
Yeah, I like this naming.